lib/entry.js

```diff
@@ -119,14 +123,33 @@ function SalesforceEntity(callScope) {
     let hasMorePages = true;
     let lastSeenTime = snapshot.previousLastModified;
 
+    // the query for the object and all its linked parent objects
+    let selectedObjects = '*';
+    selectedObjects += linkedObjects.reduce((query, obj) => {
+      if (!obj.startsWith('!')) return `${query}, ${obj}.*`;
+      return query;
+    }, '');
```

Comment:
Why not use `const` and have `'*'` be the second argument to the reduce function?